text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/resolver/Formatter.java

```diff
@@ -118,4 +120,22 @@ private Formatter() {
       return Tag.inserting(context.deserialize(choiceFormat.format(number)));
     }));
   }
+
+  /**
+   * Creates a replacement that inserts a progress bar. The component will be formatted by the provided fill and remaining colors and length
+   *
+   * <p>This tag expects a text to use to denote progress and can accept length, filled color and empty color. The text has to be provided as first argument.
+   *    In case the fill and empty colors and length are not provided, they will default to {@link ProgressTag#DEFAULT_FILLED_COLOR}, {@link ProgressTag#DEFAULT_REMAINING_COLOR} and {@value ProgressTag#DEFAULT_LENGTH} respectively.</p>
+   *
+   * <p>This replacement is auto-closing, so its style will not influence the style of following components.</p>
+   *
+   * @param key the key
+   * @param progressSupplier the supplier for the progress value
+   * @return the placeholder
+   * @since 4.11.0
```

Comment:
```suggestion
   * @since 4.13.0
```

unlikely to make it in any earlier than that